README.md

```diff
@@ -76,12 +76,9 @@ initializePhraseAppEditor(config);
 
 #### Using the US Datacenter with ICE
 
-In addition to `phraseEnabled`, `projectId` and `accountID` in the config, also add the US specific URLs to enable working through the US endpoint.
```

Comment:
Question - why is `accountID` removed from the description here? The config above on Lines 66-70 has it there 🤔 

If we don't want to list these properties, we could adjust this line to `In addition to the settings in your config, set the US datacenter to enable it working with the US endpoints.`